cmd/cli/client/root.go

```diff
@@ -19,7 +19,6 @@ var (
 func init() {
 	Cmd.PersistentFlags().String("node-url", config.DefaultPDPClient.NodeURL, "URL of a Piri node")
 	cobra.CheckErr(viper.BindPFlag("node_url", Cmd.PersistentFlags().Lookup("node-url")))
-	cobra.CheckErr(Cmd.MarkPersistentFlagRequired("node-url"))
```

Comment:
No longer require the flag as a working default is already present in the config